pkg/sql/schemachanger/testdata/explain/create_sequence

```diff
@@ -0,0 +1,130 @@
+/* setup */
+
+/* test */
+EXPLAIN (ddl) CREATE SEQUENCE sq1 MINVALUE 1 MAXVALUE 9223372036854775807 INCREMENT 1 START 32;
+----
+Schema change plan for CREATE SEQUENCE ‹defaultdb›.‹public›.‹sq1› MINVALUE 1 MAXVALUE 9223372036854775807 INCREMENT 1 START 32;
+ ├── StatementPhase
+ │    └── Stage 1 of 1 in StatementPhase
+ │         ├── 18 elements transitioning toward PUBLIC
+ │         │    ├── ABSENT → PUBLIC Sequence:{DescID: 104 (sq1+)}
+ │         │    ├── ABSENT → PUBLIC SequenceOption:{DescID: 104 (sq1+), Name: "INCREMENT"}
+ │         │    ├── ABSENT → PUBLIC SequenceOption:{DescID: 104 (sq1+), Name: "MINVALUE"}
+ │         │    ├── ABSENT → PUBLIC SequenceOption:{DescID: 104 (sq1+), Name: "MAXVALUE"}
+ │         │    ├── ABSENT → PUBLIC SequenceOption:{DescID: 104 (sq1+), Name: "START"}
+ │         │    ├── ABSENT → PUBLIC SequenceOption:{DescID: 104 (sq1+), Name: "VIRTUAL"}
+ │         │    ├── ABSENT → PUBLIC SequenceOption:{DescID: 104 (sq1+), Name: "CACHE"}
```

Comment:
This is out of scope for this PR but come to think of it, we'll also have to bind the value to a new `Value` attr in screl at some point, to handle `ALTER SEQUENCE` DDLs.